packages/stateless/components/proposal/MultipleChoiceOption.tsx

```diff
@@ -0,0 +1,230 @@
+import { Close, CopyAllOutlined } from '@mui/icons-material'
+import CircleIcon from '@mui/icons-material/Circle'
+import cloneDeep from 'lodash.clonedeep'
+import {
+  Control,
+  FieldArrayMethodProps,
+  FieldErrors,
+  FieldValues,
+  Path,
+  UseFormRegister,
+  useFieldArray,
+  useFormContext,
+} from 'react-hook-form'
+import { useTranslation } from 'react-i18next'
+
+import { SuspenseLoader } from '@dao-dao/stateful'
+import { NewProposalForm } from '@dao-dao/stateful/proposal-module-adapter/adapters/CwdProposalMultiple/types'
+import {
+  Action,
+  ActionKey,
+  ActionKeyAndData,
+  UseDefaults,
+  UseTransformToCosmos,
+} from '@dao-dao/types'
+import { validateRequired } from '@dao-dao/utils'
+
+import { ActionCardLoader, ActionSelector } from '../actions'
+import { Button } from '../buttons/Button'
+import { InputErrorMessage } from '../inputs/InputErrorMessage'
+import { TextAreaInput } from '../inputs/TextAreaInput'
+import { TextInput } from '../inputs/TextInput'
+
+export interface MultipleChoiceOptionData {
+  title: string
+  description: string
+  actionData: ActionKeyAndData[]
+}
+
+export interface MultipleChoiceOptionProps<
+  FV extends FieldValues,
+  FieldName extends Path<FV>
+> {
+  errorsOption: FieldErrors
+  title: FieldName
+  description: FieldName
+  registerOption: UseFormRegister<FV>
+  index: number
+  control: Control<FV>
+  actions: Action[]
+  removeOption: () => void
+  addOption: (
+    value:
+      | Partial<MultipleChoiceOptionData>
+      | Partial<MultipleChoiceOptionData>[],
+    options?: FieldArrayMethodProps | undefined
+  ) => void
+  actionsWithData: Partial<
+    Record<
+      ActionKey,
+      {
+        action: Action
+        transform: ReturnType<UseTransformToCosmos>
+        defaults: ReturnType<UseDefaults>
+      }
+    >
+  >
+}
+
+export const MultipleChoiceOption = <
```

Comment:
since this only makes sense in the context of the proposal module adapter (i.e. it's importing stuff from the stateful adapter), it is probably better to define this in that adapter (in the `ui` folder). in general the `stateless` package cannot import from the `stateful` package or we'll have cyclic dependencies